### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard: User have to 'TAB' twice to move over the controls such as 'Upload' button and 'Select All/None' checkbox.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4009]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
4. Navigate to 'Upload' Button and 'Select All/None' Checkbox using TAB.
5. Verify how Focus moves using TAB when moving over these controls.

**Actual:**
User have to 'TAB' twice to move over these controls. i.e. focus remains same after first TAB.

Same issue observed throughout the application on two "header" controls.

**Expected:**
Single 'TAB' should be enough to move to the next controls present in the sequence.

**User Impact:**
Keyboard user will get confuse while moving over these controls as he would have to 'TAB' twice to move over these controls.

**Attachment:**
(https://github.com/jupyter/notebook/files/2401236/A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
